### ISSUE TITLE
Please add link to an example list of supported LED bulbs/light hardware to FAQ

### ISSUE THREAD

**Issue by rsnodgrass** [claytonjn/hass-circadian_lighting#52]
Would be really useful for people looking at this to get a sense of what bulbs/hardware might be supported by this to enable quickly researching upgrade options.  Thanks!

**Comment by claytonjn** [comment]
As far as I know, this is the list: https://www.home-assistant.io/integrations#light

Are there lights that are not supported that I'm unaware of?

**Comment by rsnodgrass** [comment]
That list is a wide variety of light interfaces where the underlying
hardware may or may not support setting of the RGB.

I was curious of a sample list of actual lighting hardware that others have
used successfully with HASS Circadian Lighting.  For instance:

* HALO RL4069BLE40AWH Smart Bluetooth LED Recessed Light

**Comment by rsnodgrass** [comment]
E.g. answering the question for a complete newbie:  Love the idea, want to
integrate into HASS, but what are some examples of things I may need to
purchase.  Get them started down the path quickly of figuring out their
plan.  Thanks.

**Comment by claytonjn** [comment]
> That list is a wide variety of light interfaces where the underlying hardware may or may not support setting of the RGB.

CL works with color temp, RGB, or just dimming lights. If a user has/buys a light that's just on/off I don't know what they would expect CL to do. Also, I don't know of any connected lights that are only on/off (I'm sure they exist).


> * HALO RL4069BLE40AWH Smart Bluetooth LED Recessed Light

That would not work, based on https://www.home-assistant.io/integrations#light. There is no Bluetooth light support, nor is there HALO specific device support.


All that being said, I'm perfectly fine with having a list of supported devices and I do think it's a fine idea. But I have pretty much exclusively Hue products so it would be up to the community to populate a list. I think the best place for that would be the Wiki of this repo, which anyone should be able to contribute to.

**Comment by rsnodgrass** [comment]
Agreed...it would be community maintained and obviously non-complete, but a
good starting point for research.  wiki is perfect.

Good to know about the dimmer capabilities!  I didn't know this worked with
just dimmable switches...installing in two different HASS installations
tonight!

**Comment by ahartman** [comment]
Easiest to use in my opinion is the Tradfri range of IKEA. They come in 3 flavours, all dimmable:
= dimmable white
= dimmable and adjustable white colour temperature
= dimmable and full RGB

They are not expensive and - at least the many in my house - absolutely reliable.
Regards, ahartman, belgium

**Comment by jamman9000** [comment]
Even a better explanation and/or one example of the type of bulb that fits each "Switch Configuration Variable" would be very helpful. For example, google just told me what "mireds" means but heck if I know which bulb needs that... no idea what "XY" is either. The other two are kind of self-explanatory. The only smart bulbs I have are Wyze and a semi-smart IR controlled cheapo. I'll probably have to try each config to see which works for Wyze.

**Comment by claytonjn** [comment]
The best way to figure out out is opening the light entity in the frontend. 

![image](https://user-images.githubusercontent.com/3850252/72575829-c25a3700-389b-11ea-83b1-d8d57016b1d3.png) - If it has this section, use lights_ct

![image](https://user-images.githubusercontent.com/3850252/72575898-03524b80-389c-11ea-809f-4143c573374b.png) - If it doesn't have the above section but does have this section, use lights_rgb or lights_xy (really just have to test)

![image](https://user-images.githubusercontent.com/3850252/72575929-20871a00-389c-11ea-8737-8066f422019b.png) - If it only has this section, use lights_brightness

When I get a change I'll add the above to the wiki.

**Comment by bryshaw** [comment]
> The best way to figure out out is opening the light entity in the frontend.
> 
> ![image](https://user-images.githubusercontent.com/3850252/72575829-c25a3700-389b-11ea-83b1-d8d57016b1d3.png) - If it has this section, use lights_ct
> 
> ![image](https://user-images.githubusercontent.com/3850252/72575898-03524b80-389c-11ea-809f-4143c573374b.png) - If it doesn't have the above section but does have this section, use lights_rgb or lights_xy (really just have to test)
> 
> ![image](https://user-images.githubusercontent.com/3850252/72575929-20871a00-389c-11ea-8737-8066f422019b.png) - If it only has this section, use lights_brightness
> 
> When I get a change I'll add the above to the wiki.

I too had head scratching moments with setup regarding what to use. I used RGB for my Hue Color lights and the lights had a pinkish hue to them. Is that not ideal? Should I stick with CT? I was under the impression that the reason it was a pinkish hue was to get it more inline with circadian values.
